docs/src/main/tut/typeclasses/reducible.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: docs
+title:  "Reducible"
+section: "typeclasses"
+source: "core/src/main/scala/cats/Reducible.scala"
+scaladoc: "#cats.Reducible"
+---
+
+# Reducible
+
+`Reducible` extends the `Foldable` type class with additional `reduce` methods.
+
+You may have come by one of the `reduce`, `reduceLeft` or `reduceOption` defined in Scala's standard collections.
+`Reducible` offers exactly these methods with the guarantee, that the collection won't throw an exception at runtime, without having to reduce to an `Option`.
```

Comment:
Minor: I don't think that you need a comma after "guarantee".

Reply:
I guess that years of Java have made me paranoid. Maybe "the collection won't throw an exception at runtime" is a bit of a strong guarantee. Maybe we should just say that these methods won't throw an exception due to a collection being empty? :)